phpseclib/File/X509.php

```diff
@@ -4407,7 +4402,7 @@ function unrevoke($serial)
      */
     function getRevoked($serial)
     {
-        if (is_array($rclist = $this->_subArray($this->currentCert, 'tbsCertList/revokedCertificates'))) {
+        if (is_array($rclist = &$this->_subArray($this->currentCert, 'tbsCertList/revokedCertificates'))) {
```

Comment:
This is triky: check this link for general background http://jpauli.github.io/2014/06/27/references-mismatch.html